### PR TITLE
Clean up codebase

### PR DESCRIPTION
**Pull request by HannesTuhkala**

### CODE REVIEW

**Review by Emiluren — APPROVED** [review]

File: src/server.rs [hunk 1]
```diff
@@ -76,13 +74,21 @@ impl Server {
...
+        }
+
+        constants::DELTA_TIME
+    }
+
+    fn slowtime_is_active(&mut self) -> bool {
         for player in &mut self.state.players {
             if player.powerups.iter().any(|powerup|powerup.kind == PowerUpKind::SlowTime) {
-                return 1./300.;
+                true;
```
Comment (malcx95) [inline src/server.rs]: If you put a semicolon after the value you want to return (without expicitly writing return), it won't return it, right @TheZoq2 ?

Comment (Emiluren) [reply src/server.rs]: No it won't

Comment (malcx95) [reply src/server.rs]: Then this should be fixed.

**Review by malcx95 — CHANGES_REQUESTED** [review]

File: src/server.rs (hunk 1 above)
Comment (Emiluren) [reply src/server.rs]: It looks like `mut` could be removed also

File: src/server.rs [hunk 2]
```diff
@@ -76,13 +74,21 @@ impl Server {
     }
 
     fn get_delta_time(&mut self) -> f32 {
```
Comment (malcx95) [inline src/server.rs]: Remove mut

**Review by malcx95 — CHANGES_REQUESTED** [review]

File: src/server.rs (hunk 1 above)
Comment (malcx95) [reply src/server.rs]: Yes, and then `mut` can be removed from `get_delta_time` as well

File: libplen/src/constants.rs [hunk 3]
```diff
@@ -9,6 +9,7 @@ pub const POWERUP_BOUNCE_HEIGHT: f32 = 10.;
 
 // currently hardcoded to the background image size
 pub const WORLD_SIZE: f32 = 3000.;
+pub const DELTA_TIME: f32 = 0.01;
```
Comment (TheZoq2) [inline libplen/src/constants.rs:12]: Sleep duration [here](https://github.com/HannesTuhkala/plen/blob/master/src/server.rs#L91) should probably also use this

Comment (HannesTuhkala) [reply libplen/src/constants.rs:12]: Fixed in next commit

**Review by malcx95 — APPROVED** [review]

**Pull request merged**